### PR TITLE
first pass endpoints vignette (for closing #96) (take 2)

### PR DESCRIPTION
**Pull request by dsweber2**
See #102 for actual discussion, this should be ready to merge as is.

### CODE REVIEW

File: R/endpoints.R [hunk 1]
```diff
@@ -1190,7 +1204,7 @@ pvt_ght <- function(auth, locations, epiweeks, query) {
 #' TODO: find a non-trivial region
 #' @examples
 #' \dontrun{
-#' kcdc_ili(regions = "?", epiweeks = epirange(201201, 202001)) %>% fetch()
+#' kcdc_ili(regions = "ROC", epiweeks = 200436) %>% fetch()
```
Comment (dshemetov) [inline R/endpoints.R:1207]:
```suggestion
#' kcdc_ili(regions = "ROK", epiweeks = 200436) %>% fetch()
```

File: R/endpoints.R [hunk 2]
```diff
@@ -738,7 +748,7 @@ covidcast <- function(
 #'
 #' @examples
 #' \donttest{
-#' delphi(system = "ec", epiweek = 201501) %>% fetch()
+#' delphi(system = "ec", epiweek = 202006) %>% fetch()
```
Comment (dshemetov) [inline R/endpoints.R:751]:
```suggestion
#' delphi(system = "ec", epiweek = 201501) %>% fetch()
```

**Review by dshemetov — CHANGES_REQUESTED** [review]
Few typos in docs, otherwise we're ready to merge 🚀 🚀 🚀

**Comment by dsweber2** [comment]
and done

**Comment by brookslogan** [comment]
Closes #96.

**Review by brookslogan — APPROVED** [review]
Changes were already approved in #102 + a few typos and a compatibility update have been addressed.

**Review by dshemetov — APPROVED** [review]

**Pull request merged**